slick/src/main/scala/slick/ast/Comprehension.scala

```diff
@@ -9,9 +9,11 @@ final case class Comprehension(sym: TermSymbol, from: Node, select: Node, where:
                                groupBy: Option[Node] = None, orderBy: ConstArray[(Node, Ordering)] = ConstArray.empty,
                                having: Option[Node] = None,
                                distinct: Option[Node] = None,
-                               fetch: Option[Node] = None, offset: Option[Node] = None) extends DefNode {
+                               fetch: Option[Node] = None,
+                               offset: Option[Node] = None,
+                               forUpdate: Option[Node] = None) extends DefNode {
```

Comment:
Why is forUpdate not a Boolean? It doesn't look like any of the code generators uses the contents, nor does `mergeToComprehensions` ensure that it's kept consistent.